wurst/_handles/Trigger.wurst

```diff
@@ -1,5 +1,13 @@
 package Trigger
 import NoWurst
+import Player
```

Comment:
The changes you made here overlap with stuff from RegisterEvents

Reply:
Oh, thanks, did not realize it was there. Thought it'd be intuitive to have those extension functions for Trigger.